src/components/NewsletterForm/NewsletterForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+
+import useScript from './NewsletterForm.script';
+import {
+  EmailInput,
+  EmbedForm,
+  MainForm,
+  NewsletterFormBody,
+  RegistrationConfirmation,
+  Sub,
+  SubmitButton,
+} from './NewsletterForm.styled';
+import type { NewsletterForm as NewsletterFormType } from './NewsletterForm.types';
+
+export const NewsletterForm: NewsletterFormType = () => {
+  useScript('https://www.google.com/recaptcha/api.js');
+  useScript(
+    'https://groot.mailerlite.com/js/w/webforms.min.js?v2d8fb22bb5b3677f161552cd9e774127',
```

Comment:
v2d8fb22bb5b3677f161552cd9e774127 jesli to jest klucz - to samo, trzymasz w env

Reply:
zrobione